### DISCUSSION TITLE
Unload model

### DISCUSSION THREAD

**Discussion by nixsrv** [SYSTRAN/faster-whisper#431] [Q&A] [answered]
Hi! How to unload faster-whisper model from VRAM? I need to use another model(NER) after work of F-W.

**Comment by guillaumekln** [comment, marked as answer]
You could delete the model object and then force the  Python garbage collector to run. Try this:

```python
import gc
from faster_whisper import WhisperModel

model = WhisperModel("large-v2")
del model
gc.collect()
```

**Comment by nixsrv** [comment]
Thanks. That's work.

**Comment by Denis-Kazakov** [comment]
Don't I also have to run torch.cuda.empty_cache()?
